generatebundlefile/scripts/generate_bundle.sh

```diff
@@ -27,23 +27,24 @@ BASE_DIRECTORY=$(git rev-parse --show-toplevel)
 chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile 
 
 # Faster way to install Cosign compared to go install github.com/sigstore/cosign/cmd/cosign@v1.5.1
-curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
+COSIGN_URL=$(curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
```

Comment:
This is fine, I guess, but if you have access to the `jq` program, and you probably do, then something like this might be better:

```sh
curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
     | jq -r '.assets[].browser_download_url | select(endswith("cosign-linux-amd64"))'
```